charts/common/values-ci-tests.yaml

```diff
@@ -37,5 +37,8 @@ container:
       failureThreshold: 6
       periodSeconds: 5
 
+postgres:
+  enabled: true
+
```

Comment:
I think this is best overriden in tests that tries to verify postgres behaviour.
Should probably not be enabled for all tests.

Reply:
Probably explains why the install step fails as well 👍🏻